changelogs/unreleased/protobuf_encoder_module.md

```diff
@@ -0,0 +1,3 @@
+## feature/lua
```

Comment:
> Since: 3.1

3.1 feature window is closed now (`3.1.0` is already released).

I would just drop this line. We can add an actual information about releases into the documentation issue, when the feature lands into tarantool.

Please, update commit message and PR description.

Reply:
Seems solved. Please, respond to comments, when they're fixed.